content/docs/intro/console/accounts-and-organizations/organizations.md

```diff
@@ -128,24 +128,25 @@ The Organization drop-down list displays all of the organizations your account i
 associated with, and lets you add a new organization backed by a third
 party identity provider. To switch to a different organization:
 
-1. Click on the organization drop-down list on the upper left corner of the Console screen
+1. Select the organization drop-down list on the upper left corner of the Console screen
 next to the Pulumi logo.
 1. Select your organization name. The organization page view is dependent on your role within the Pulumi organization.
 
 <img class="lg:max-w-xl" src="/images/docs/reference/service/organization-view.png" alt="Stack Outputs and Configuration">
 
   _Members_. Pulumi organization members only see the Stacks and People tabs.
 
-  _Admins_. Pulumi organization administrators see the Stacks, People, Teams, Webhooks,
+  _Admins_. Pulumi organization admins see the Stacks, People, Teams, Webhooks,
 and Settings tabs.
 
 | Console Tab | Description |
 |--------|--------|
-| Stacks | A searchable list of organization stacks that you can group by project and tag. See [Project and Stack Management]({{< relref "/docs/intro/console/collaboration/project-and-stack-management">}}) to learn more. |
-| People | A list of active members of the Pulumi organization. |
+| Dashboard | An overview of the organization including recently updated stacks, recent activiity, and a resource count graph. |
+| Projects | A searchable list of organization stacks that you can group by project and tag. For more information, see [Project and Stack Management]({{< relref "/docs/intro/console/collaboration/project-and-stack-management">}}). |
+| Members | A list of active members of the Pulumi organization. |
 | Teams | A [Team Pro]({{< relref "/pricing" >}}) feature that provides a way to assign stack permissions to groups of organization members. |
-| Webhooks | A [Team Pro]({{< relref "/pricing" >}}) feature that allows external services to be notified about events happening on an organization, including events occurring on organization stacks. |
-| Settings | A place for you to manage your organization settings including subscription and payment information and history, base permissions to all of the organization's stacks (including stack creation and deletion), and links to Pulumi's [continuous delivery guides]({{< relref "/docs/guides/continuous-delivery" >}}). |
+| Policies | Policies empower you to set guardrails to enforce best practices and compliance. |
```

Comment:
This one's a bit weird in that the others explain what the tab is, whereas this one explains the concept of policies. Seems like it's missing something before it, like

```suggestion
| Policies | Lists of organization policies and policy groups. Policies allow you to set guardrails to enforce best practices and compliance. |
```

Reply:
heh yah thx for fixing it, i just pulled that text from the ui